site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md

```diff
@@ -0,0 +1,596 @@
+</figure>
+
+{% Details %}  
+{% DetailsSummary 'h3' %}  
+How it works in detail: event-level reports  
+{% endDetailsSummary %}  
+Ad links can be configured with attributes that are specific to ad conversions:
+
+- Custom data to attach to an ad click (or view) on the publisher's side, for example a
+  click ID or campaign ID.
+- The site for which a conversion is expected for this ad.
+- The reporting endpoint that should be notified of successful conversions, that is,
+  receive the reports.
+- The cut-off date for when conversions can no longer be counted for this ad.
+
+Note: it's also possible to register an attribution source for navigations [initiated by
+`window.open()`](https://github.com/WICG/conversion-measurement-api/blob/main/event_attribution_reporting_clicks.md#registering-attribution-sources-for-windowopen-navigations)
+or, for views, via a [JavaScript
+API](https://github.com/WICG/conversion-measurement-api/blob/main/event_attribution_reporting_views.md#registering-attribution-sources-with-javascript).
+
+When the user clicks or sees a specially configured ad, the browser—on the user's local
+device—records this event, alongside the attribution configuration data that was
+specified.
+
+Later on, the user visits the advertiser's website and performs an action that the
+advertiser or their adtech provider categorizes as a conversion, such as a purchase. When
+this happens, the advertiser or adtech provider triggers an attribution: it asks the
+browser to record a conversion with a certain value `trigger-data`, and the ad click (or
+view) and conversion event are matched by the user's browser.
+
+The browser finally schedules a report to be sent to the endpoint specified on the ad
+side. This report includes:
+
+- Custom ad-side data that was attached to the ad click or view that led to this
+  conversion.
+- Custom conversion-side data, with some noise.
+
+If several conversions are registered for a given ad click (or view), corresponding
+reports are scheduled to be sent. A single report can be sent for views, and up to three
+reports for clicks.
+
+Reports are sent by the browser after a delay⏤days or sometimes weeks after a conversion.
+
+{% endDetails %}
+
+### Aggregate reports
+
+<figure>
+{% Img src="image/O2RNUyVSLubjvENAT3e7JSdqSOx1/HAl0ppkoxoGCtttWDk2A.png", alt="ALT_TEXT_HERE", width="800", height="1140" %}
+  <figcaption>Aggregate reports are generated as follows: the browser matches detailed clicks or views ("attribution source events") with detailed conversion data ("attribution trigger data") defined by an adtech. Adtech-defined code runs in a worklet to define contributions that will be sent by the browser in order to be used to compute aggregate reports. Aggregation services are responsible for privately computing aggregate reports for adtech.</figcaption>
```

Comment:
I suggest pulling these descriptions out of the caption element, and into paragraph text. The longer captions (below) don't look good with centred formatting.